codespell_lib/data/dictionary.txt

```diff
@@ -14979,6 +14986,7 @@ unihabited->uninhabited
 unilateraly->unilaterally
 unilatreal->unilateral
 unilatreally->unilaterally
+uninitalize->uninitialize
```

Comment:
Can you add the s versions of all of these.

Reply:
Done

Reply:
Sorry @luzpaz , I wasn't very clear, I meant the British versions, i.e. uninitialise and friends.

Reply:
LOL, a-ok, I think I got it this time :rofl: f652e58